lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -615,15 +615,15 @@ def _build_problem_list(cls, course_blocks, root, path=None):
             Tuple[str, List[str], UsageKey]: tuple of a block's display name, path, and
                 usage key
         """
-        display_name = course_blocks.get_xblock_field(root, 'display_name')
+        name = course_blocks.get_xblock_field(root, 'display_name') or root.category
```

Comment:
@ormsbee I'm making it fall back to the 'category' in case there is no display name. I hope that works.